internal/test-charm-repo/bundle/openstack/README.md

```diff
@@ -36,11 +36,11 @@ The OpenStack cloud deployed is completely clean; the charms don't attempt to co
 Niggles
 -------
 
-The neutron-gateway service requires a service unit with two network interfaces to provide full functionality; this part of OpenStack provides L3 routing between tenant networks and the rest of the world.  Its possible todo this when testing on OpenStack by adding a second network interface to the neutron-gateway service:
+The neutron-gateway application requires an application unit with two network interfaces to provide full functionality; this part of OpenStack provides L3 routing between tenant networks and the rest of the world.  Its possible todo this when testing on OpenStack by adding a second network interface to the neutron-gateway application:
```

Comment:
not sure that we should be changing this file, since it's taken from a real bundle...